akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1627,17 +1627,36 @@ trait DiagnosticLoggingAdapter extends LoggingAdapter {
 
 /** DO NOT INHERIT: Class is open only for use by akka-slf4j*/
 @DoNotInherit
-class LogMarker(val name: String)
+class LogMarker(val name: String, val properties: Map[String, Any]) {
```

Comment:
LogMarker expanded with additional properties, which will be emitted as MDC values by the Slf4jLogger

Reply:
We could expand our classic logging API even further, or construct the `LogMarker`/ log message with some dsl, but since we are the "only" users I'm not sure it's worth it. End users should use that Logstash api if they like.

We can't use Logstash api directly since we don't want that dependency from akka-actor, and classic logging is based on the eventStream.

Yes, they end up as MDC because that is automatically handled by logstash encoder. Otherwise we would have to provide a converter to logstash encoder, and end result would anyway be the same.

Reply:
So if we'd want to emit structured logging from any typed actor (receptionist?) we'd just use MDC directly?

Reply:
We could do that, or just use the classic logging for those.